providers/shared/components/datafeed/id.ftl

```diff
@@ -56,6 +56,30 @@
                         "Type" : STRING_TYPE,
                         "Description" : "The prefix appeneded to the object name when processing failed",
                         "Default" : "error/"
+                    },
+                    {
+                        "Names" : "Include",
```

Comment:
Will need the same on other sources of logging? If so, maybe worth making a separate configuration variable that can be used elsewhere?

Reply:
Its really only optional here when you are using a datafeed as an application component where it might not be useful

Reply:
OK - fair call.